src/PrestaShopBundle/Resources/config/services/bundle/routing.yml

```diff
@@ -53,3 +53,9 @@ services:
         class: 'PrestaShopBundle\Routing\Linter\AdminRouteProvider'
         arguments:
             - '@router'
+
+    prestashop.bundle.routing.controller_stack:
+        class: 'PrestaShopBundle\Routing\ControllerStack'
+
+    controller_stack:
```

Comment:
This alias allows module developers to simply retrieve this doing `$this->getContainer()->get('controller_stack');` (there are similar Symfony aliases : `router`,  `request_stack` ...)